index.js

```diff
@@ -36,14 +36,41 @@ export async function acs (pubSubMessage, context) {
   })
 
   try {
-    const csvData = new Audience(pubSubMessage.data).acsCsv
-    await s3.upload({
-      Bucket: process.env.S3_BUCKET,
-      Key: `tealium_to_acs_${new Date().toISOString().slice(0,10)}.csv`,
-      Body: csvData,
-    }, (err, data) => {
-      console.log(`File uploaded successfully to ${data.Location}`)
-    })
+    await s3.getObject(
+      { Bucket: process.env.S3_BUCKET, Key: "tealium_to_acs.csv" },
```

Comment:
Also changed the name of the file to just ```tealium_to_acs.csv```, as there should always only be one file with this name in the bucket at any given time.

Reply:
You'll need to prefix this key as well since the bucket is just `cru-sftp` now.
```suggestion
      { Bucket: process.env.S3_BUCKET, Key: "adobeftp/Tealium/tealium_to_acs.csv" },
```